cron/daily.php

```diff
@@ -19,25 +19,8 @@
  */
 define('CRON_MODE', true);
 require_once(dirname(__DIR__) . DIRECTORY_SEPARATOR . "config.php");
+require_once __DIR__ . '/../vendor/autoload.php';
```

Comment:
Why do you need to include here the autoload?
Doesn't it work without it?

Reply:
Because classes are autoloaded by composer.

Reply:
Yes, but the config already includes the autoload file so this include is redundant.

Reply:
Really? I miread that maybe, or I expected config should be only config. My example (for phpstan) config does not include that file...